### PR TITLE
Support logLevel for job master/worker

### PR DESCRIPTION
**Pull request by jiacheliu3**
`logLevel` command currently only works for master and workers. This change enables the command to set log level on job master/workers.

### CODE REVIEW

File: shell/src/main/java/alluxio/cli/LogLevel.java [hunk 1]
```diff
@@ -192,6 +236,7 @@ private static void setLogLevel(final TargetInfo targetInfo, String logName, Str
     if (level != null) {
       uriBuilder.addParameter(LEVEL_OPTION_NAME, level);
     }
+    LOG.info("Setting log level on %s%n", uriBuilder.toString());
```
Comment (gpang) [inline shell/src/main/java/alluxio/cli/LogLevel.java:239]: Did you want this for log, or for System.out?

**Review by gpang — COMMENTED** [review]
@jiacheliu3 Thanks!

File: shell/src/main/java/alluxio/cli/LogLevel.java (hunk 1 above)
Comment (jiacheliu3) [reply shell/src/main/java/alluxio/cli/LogLevel.java:239]: I think users need not know the exact URL unless something goes wrong, then they can find it in the log. So I'd like this to be in the log.

**Review by gpang — APPROVED** [review]
LGTM

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Pull request merged**